### DISCUSSION TITLE
search for torrent with api issue

### DISCUSSION THREAD

**Discussion by itsjusttestingemail** [anasty17/mirror-leech-telegram-bot#1203] [Q&A] [answered]
Hello
i'm facing a problem with /search command when i use that command to find content directly on torrent using api plugins then i'm getting that error in response

**No API link or search PLUGINS added for this function**

but plugins are already added on config.env

please help me to solve this isssue

**Comment by anasty17** [comment]
Check of really added from bot settings. If database added then u should add them from bot not by editing config.env

**Reply by pakbottest** [reply]
i added commands from bot settings. Seacrh function is working fine for couple of days but when i updated bot and rebuild it again now search function not working.

**Reply by anasty17** [reply]
Commands! From bot settings!!!

**Reply by itsjusttestingemail** [reply]
brother could you please tell me what should i do to solve this error. I tested it without database and still same error.

**Comment by anasty17** [comment, marked as answer]
Fixed update ur repo

**Reply by itsjusttestingemail** [reply]
ok bro going to update will tell you after check

**Reply by itsjusttestingemail** [reply]
Thanks Bro now working fine
